README.md

```diff
@@ -8,7 +8,7 @@ Currently it supports the following:
 
 - [Ubuntu Extended Security Maintenance](https://ubuntu.com/esm) archive.
 - [Canonical Livepatch](https://www.ubuntu.com/server/livepatch) service for managed live kernel patching.
-
+- [Canonical FIPS 140-2 Certified Modules] Install Configure and Enable FIPS modules.
```

Comment:
This is missing a link to a web page

Reply:
ok, used "dpkg-query -s" instead. 
Unfortunately, we do not have a web page yet for FIPS. As soon as we do, I will add it here.

Reply:
I will remove the reference from here until we do get a web page

Reply:
hmmmm... is it ok to leave the reference showing the script supports this? Or should i remove it until we get a web page?

Reply:
You can leave the reference to FIPS, but remove the brackets until there's a link (otherwise they show in the output).
I'd just write it as:

- Canonical FIPS 140-2 Certified modules management